### DISCUSSION TITLE
Metadata Synchronization With PBI service issue.

### DISCUSSION THREAD

**Discussion by PoojaReddyM96** [TabularEditor/TabularEditor3#1333] [🙋♀️TE3 Q&A]
Hi Everyone,

We have a dataset that is connected to Tabular Editor 3. I am currently using 3.15.0 Business Edition. We make all our dataset related changes in Tabular Editor since we have XMLA enabled. It was all working fine until 2 days ago but since then we are getting the below error whenever we are trying to make any changes to the dataset. We are unable to edit the code and make changes, update schemas and save it. However, we are able to refresh the tables and partitions.

We also tried deploying this dataset with a new name and there we are able to comment, uncomment, update the schemas. It is only with this dataset that we are having an issue.

Did anyone face a similar issue or do anyone have any idea on how to resolve it? 

We cannot re publish the dataset with a new name as this is our production dataset and it is also being used as a connection string by many users in several files so this would change the link and we cannot afford that.

Any help would be appreciated.

Thank you.
<img width="491" alt="Save Issue" src="https://github.com/TabularEditor/TabularEditor3/assets/173822788/9a26b1d0-d519-4e27-bae7-d134fa948cdf">

**Comment by otykier** [comment]
This error message comes from the XMLA endpoint. I suggest you open a support ticket with Microsoft. Make sure to include the RootActivityId, since that will allow their support engineers to find the trace logs for this operation.

**Reply by PoojaReddyM96** [reply]
Thank you so much for your reply @otykier.
We have already raised a Microsoft support request and that is in progress. They are still investigating the issue.
We have also received the trace logs from them for the operation.
I can share the log files here with you if you can help us on that.
Thank you.

**Reply by PoojaReddyM96** [reply]
Hi @otykier ,
Please find the trace logs attached.
Looking forward to hearing from you.
Thank you.
[Book1.xlsx](https://github.com/user-attachments/files/16011694/Book1.xlsx)

**Reply by PoojaReddyM96** [reply]
Hi @otykier,
We have also received an update from the Microsoft Internal team that - **Since Tabular Editor is a 3rd Party Tool they do not investigate any activities related to it.**
Now we are stuck since that is our production dataset and we are unable to make any changes to it hence reaching out to you for quick help.
We would appreciate any kind of support and help.
Thank you.

**Reply by otykier** [reply]
Hi @PoojaReddyM96 
Sorry to hear that. Kindly send me the support ticket # for the ticket you raised with Microsoft. Then I'll see what I can do. daniel@tabulareditor.com

**Reply by PoojaReddyM96** [reply]
Hi @otykier,
Thank you for the quick response. 
Here is the Support Request Number - **2406200050000061**

**Reply by otykier** [reply]
Hi @PoojaReddyM96 
Thanks. Is the problematic model a Direct Lake model by any chance?

**Reply by PoojaReddyM96** [reply]
Hi @otykier,
No, all our dim and fact tables are in import mode. Our model is composite since we have one table whose storage type is Direct Query.
Our reports are then connected live to this semantic model.
